tests/functional/disaster-recovery/regional-dr/conftest.py

```
@@ -449,6 +450,70 @@ def teardown():
     return factory
 
 
+@pytest.fixture()
```

Ruff (0.15.20)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)